lib/protractor-testability-plugin.js

```diff
@@ -22,8 +22,24 @@ return {
                         };
                     },
                     getTestability: function () {
+                        // Allow users to set their own testability objects to
+                        // map anything Angular provides for Protractor to the
+                        // framework they're using.
+                        var whenStableDefault = function (cb) { cb(); };
+                        var customAngularTestability = window.customAngularTestability;
+                        if (customAngularTestability) {
+                            // Allow setting custom whenStable of any value if
+                            // it's assigned. If not assigned, use default.
+                            // The user may want to set whenStable to anything,
+                            // even falsey, for testing reasons, so they should
+                            // be allowed to do so.
+                            if (typeof customAngularTestability.whenStable === 'undefined') {
+                                customAngularTestability.whenStable = whenStableDefault;
+                            }
+                            return customAngularTestability;
+                        }
                         return {
-                            whenStable: function (cb) { cb(); }
+                            whenStable: whenStableDefault
```

Comment:
Yeah I agree, but I thought this way would be better for browser support: browser support can be anything because this plugin is for non-Angular apps. But I defer to your decision :)